Assets/Scripts/Utilities/Logger.cs

```diff
@@ -30,7 +30,7 @@ public static void LogException(Exception exception)
         }
     }
 
-    public static void LogAssertion(string message)
+    public static void LogAssertion(object message)
```

Comment:
If type is `object`, than parameter should be renamed.
But I think this is bad idea to use objects as Log input.
This is person who use Loger should give appropriate input.